examples/face_landmarker/ios/FaceLandmarker/ViewContoller/ViewController.swift

```diff
@@ -0,0 +1,446 @@
+
+  // MARK: View Handling Methods
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    // Create object detector helper
+    faceLandmarkerHelper = FaceLandmarkerHelper(
+      modelPath: modelPath,
+      numFaces: numFaces,
+      minFaceDetectionConfidence: detectionConfidence,
+      minFacePresenceConfidence: presenceConfidence,
+      minTrackingConfidence: trackingConfidence,
+      runningModel: runingModel,
+      delegate: self)
+
+    runningModelTabbar.selectedItem = cameraTabbarItem
+    runningModelTabbar.delegate = self
+    cameraCapture.delegate = self
+    overlayView.clearsContextBeforeDrawing = true
+  }
+  override func viewWillAppear(_ animated: Bool) {
+    super.viewWillAppear(animated)
+#if !targetEnvironment(simulator)
+    if runingModel == .liveStream && runningModelTabbar.selectedItem == cameraTabbarItem {
+      cameraCapture.checkCameraConfigurationAndStartSession()
+    }
+#endif
+  }
+
+#if !targetEnvironment(simulator)
+  override func viewWillDisappear(_ animated: Bool) {
+    super.viewWillDisappear(animated)
+    cameraCapture.stopSession()
+  }
+#endif
+
+  override var preferredStatusBarStyle: UIStatusBarStyle {
+    return .lightContent
+  }
+
+  // MARK: Storyboard Segue Handlers
+  override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+    super.prepare(for: segue, sender: sender)
+    if segue.identifier == "EMBED" {
+      inferenceViewController = segue.destination as? InferenceViewController
+      inferenceViewController?.numFaces = numFaces
+      inferenceViewController?.delegate = self
+      bottomViewHeightConstraint.constant = inferenceBottomHeight
+      bottomSheetViewBottomSpace.constant = -inferenceBottomHeight + expandButtonHeight
+      view.layoutSubviews()
+    }
+  }
+
+  // MARK: IBAction
+
+  @IBAction func addPhotoButtonTouchUpInside(_ sender: Any) {
+    openImagePickerController()
+  }
+  // Resume camera session when click button resume
+  @IBAction func resumeButtonTouchUpInside(_ sender: Any) {
+    cameraCapture.resumeInterruptedSession { isSessionRunning in
+      if isSessionRunning {
+        self.resumeButton.isHidden = true
+        self.cameraUnavailableLabel.isHidden = true
+      }
+    }
+  }
+  // MARK: Private function
+  private func openImagePickerController() {
+    if UIImagePickerController.isSourceTypeAvailable(.savedPhotosAlbum) {
+      let imagePicker = UIImagePickerController()
+      imagePicker.delegate = self
+      imagePicker.sourceType = .savedPhotosAlbum
+      imagePicker.mediaTypes = [UTType.image.identifier, UTType.movie.identifier]
+      imagePicker.allowsEditing = false
+      DispatchQueue.main.async {
+        self.present(imagePicker, animated: true, completion: nil)
+      }
+    }
+  }
+
+  private func removePlayerViewController() {
+    playerViewController.view.removeFromSuperview()
+    playerViewController.removeFromParent()
+  }
+
+  private func processVideo(url: URL) {
```

Comment:
Same as in the helper - let's add more spacing to break this up a bit, plus comments explaining what the major tasks are.